### PR TITLE
added recapitulation page after successful order

### PR DESCRIPTION
**Pull request by ivanmerx**

### CODE REVIEW

File: wordpress/wp-content/plugins/FlexPayWoo/index.php [hunk 1]
```diff
@@ -143,15 +143,21 @@ function get_return_url($order = NULL)
...
                 "priceAmount" => $order->order_total,
                 "priceCurrency" => get_woocommerce_currency(),
                 "description" => "Payment for order # " . $order->id . " on " . get_option('siteurl'),
                 "email" => $order->billing_email,
                 "type" => 'purchase',
                 "referenceID" => $order->id
-            ));
+            );
+            if ( $this->enabled_backurl == "no"){
+            	$get_order = new WC_Order($order->id);
```
Comment (PunchyRascal) [inline wordpress/wp-content/plugins/FlexPayWoo/index.php]: why `get_order`?

File: wordpress/wp-content/plugins/FlexPayWoo/index.php [hunk 2]
```diff
@@ -143,15 +143,21 @@ function get_return_url($order = NULL)
...
+				$get_order_key = $get_order->order_key;
```
Comment (PunchyRascal) [inline wordpress/wp-content/plugins/FlexPayWoo/index.php]: Setup your editor to use indentation consistently (spaces v. tabs) and format this code accordingly

File: wordpress/wp-content/plugins/FlexPayWoo/includes/settings.php [hunk 3]
```diff
@@ -10,6 +10,12 @@
         'type' => 'checkbox',
         'label' => __('Enable this checkbox to start using Verotel/CardBilling gateway.', 'woocommerce')
     ),
+    'enabled_backurl' => array(
+        'title' => __('Enable/Disable SuccessURL', 'woocommerce'),
+        'type' => 'checkbox',
+        'label' => __('Enable this checkbox to use custom success url', 'woocommerce'),
```
Comment (PunchyRascal) [inline wordpress/wp-content/plugins/FlexPayWoo/includes/settings.php]: Try to better explain, what happens when this is disabled/enabled

File: wordpress/wp-content/plugins/FlexPayWoo/index.php (hunk 1 above)
Comment (ivanmerx) [inline wordpress/wp-content/plugins/FlexPayWoo/index.php]: Done

File: wordpress/wp-content/plugins/FlexPayWoo/includes/settings.php (hunk 3 above)
Comment (ivanmerx) [inline wordpress/wp-content/plugins/FlexPayWoo/includes/settings.php]: Done

File: wordpress/wp-content/plugins/FlexPayWoo/index.php (hunk 2 above)
Comment (ivanmerx) [inline wordpress/wp-content/plugins/FlexPayWoo/index.php]: Will look for a different editor

**Pull request merged**